src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/AuthorisedService.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.ccd.document.am.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+import uk.gov.hmcts.reform.ccd.document.am.model.enums.Permission;
+
+import java.util.List;
+
+/**
+ * Case Document.
```

Comment:
Copy paste mistake, useless commented block

Reply:
We have plenty of this same copy-paste mistakes, and all such comments are already useless. pls cleanup them.